### PR TITLE
Log levels

### LINKED ISSUE #170: Log level usage
Redefine log level such as:
- INFO (-vv): easy to follow the swap with messages such as this step happen, send money here, swap complete, funds sent to
- DEBUG (-vvv): services spawned, transaction details, mempool and block seen

### LINKED ISSUE #257: log should distinguish different swapds
```
2021-11-29T13:06:30Z INFO  farcaster_node::swapd::runtime] Terminating swap<0x7c29d85b9626425b9902e0f9bc5f49387f8b53e46d0df650ea511eae04a98077
```

### LINKED ISSUE #279: Printed final monero address in logs is Stagenet
At lines https://github.com/farcaster-project/farcaster-node/blob/main/src/walletd/runtime.rs#L1126-L1127

we print the Monero corresponding address after recovering all secret, but the address is hardcoded for Stagenet.

### PR DESCRIPTION
**Pull request by h4sh3d**
Fix #279, fix #257, fix #170 

Modify logs to resolve cited issues.

- [x] Alice Maker, Bob taker logs success path (INFO level)
- [x] Bob Maker, Alice taker logs success path (INFO level)
- [x] Swap id as prefix in swapd INFO level logs

Cancel with refund/punish path have not been tested here, but I think most of the work will be reused already and it can come in a later PR.

### CODE REVIEW

File: src/swapd/runtime.rs [hunk 1]
```diff
@@ -1421,7 +1425,7 @@ impl Runtime {
                         block,
                         tx,
                     }) if self.state.swap_role() == SwapRole::Alice => {
-                        info!(
+                        trace!(
```
Comment (TheCharlatan) [inline src/swapd/runtime.rs:1428]: I'd like this on debug

Comment (h4sh3d) [reply src/swapd/runtime.rs]: Done in 60d4f8e

**Review by TheCharlatan — APPROVED** [review]
tACK

File: src/farcasterd/runtime.rs [hunk 2]
```diff
@@ -152,11 +152,12 @@ impl Stats {
     fn success_rate(&self) -> f64 {
         let Stats { success, failure } = self;
         let total = success + failure;
-        let rate = *success as f64 / (total as f64);
+        let rate = *success as f64 / (total as f64) * 100f64;
```
Comment (zkao) [inline src/farcasterd/runtime.rs:155]: if u multiple by 100 this is not a rate

File: src/service.rs [hunk 3]
```diff
@@ -111,15 +111,15 @@ pub enum ServiceId {
     #[from]
     Peer(NodeAddr),
 
-    #[display("swap<{0:#x}>")]
```
Comment (zkao) [inline src/service.rs:114]: this was one of the few places that was printing the swapd correctly, are you sure about this?

**Review by zkao — CHANGES_REQUESTED** [review]
Let me merge this one, so that it doesnt mess up with my unmerged work

File: src/farcasterd/runtime.rs (hunk 2 above)
Comment (h4sh3d) [reply src/farcasterd/runtime.rs:155]: Yes, I change this

File: src/service.rs (hunk 3 above)
Comment (h4sh3d) [reply src/service.rs:114]: The swap if can be printed with its full version in the logs one it start if you want, and we can access it with cli ls

File: src/farcasterd/runtime.rs (hunk 2 above)
Comment (h4sh3d) [reply src/farcasterd/runtime.rs]: Done in cbdf583dca7c126c8fee095ce5afe6ca3eb6cf3b

File: src/service.rs (hunk 3 above)
Comment (h4sh3d) [reply src/service.rs:114]: Done in 83619a737880eefab230b74327a8b0d7f5ae8fcd

**Pull request merged**